src/containerapp-preview/azext_containerapp_preview/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
```

Comment:
I think maybe you cab populate all test in GA container apps here to avoid breaking?